SPECS/xcp-ng-generic-lib.spec

```diff
@@ -9,6 +9,9 @@ Source0:        https://github.com/xcp-ng/xcp-ng-generic-lib/archive/v%{version}
 BuildRequires:  cmake3
 BuildRequires:  make
 BuildRequires:  gcc
+BuildRequires:  binutils-devel
+
+Requires:  binutils
```

Comment:
Add `Requires: %{name}-debuginfo = %{version}-%{release}` ?

Reply:
(with a comment that explains that you need it for displaying a proper stacktrace in the crash handler)